--- a/gobject/gclosure.c	2016-04-08 15:57:20.566030900 -0700
+++ b/gobject/gclosure.c	2016-04-08 15:56:59.220206000 -0700
@@ -755,7 +755,7 @@
 /**
  * g_closure_invoke:
  * @closure: a #GClosure
- * @return_value: (optional) (out): a #GValue to store the return
+ * @return_value: (optional) (in): a #GValue to store the return
  *                value. May be %NULL if the callback of @closure
  *                doesn't return a value.
  * @n_param_values: the length of the @param_values array
